browser/brave_content_browser_client.cc

```diff
@@ -114,12 +121,24 @@ void BraveContentBrowserClient::BrowserURLHandlerCreated(
   ChromeContentBrowserClient::BrowserURLHandlerCreated(handler);
 }
 
-bool BraveContentBrowserClient::AllowAccessCookie(const GURL& url, const GURL& first_party,
-    content::ResourceContext* context, int render_process_id, int render_frame_id) {
+bool BraveContentBrowserClient::AllowAccessCookie(const GURL& url,
+  const GURL& first_party, content::ResourceContext* context,
```

Comment:
these should be indented with 4 spaces or (preferably) one-per line justified with the first param